gabc/details.html

```diff
@@ -121,7 +121,7 @@ <h3 id="signs">Rhythmic signs</h3>
 <p>In some scores, there are sometimes two levels of horizontal episema: a short episema beneath a longer one. To put an horizontal episemus under a note, type <strong><code>&nbsp;_0&nbsp;</code></strong>. This can be combined with a normal horizontal episemus.</p>
```

Comment:
I've always wondered if "episemus" is a proper word.  In English literature, I've always seen "episema" used in both singular and plural context.  However, I can't find a definitive answer since this is an esoteric term in English.  According to http://www.latin-dictionary.net/definition/19234/episema-episematis, it's third declension, making episema the nominative singular form and episemata the nominative plural form.